spec/queries/appeals_with_cancelled_root_task_completed_dispatch_query_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+describe AppealsWithCancelledRootTaskCompletedDispatchQuery, :postgres do
+  let!(:dispatched_appeal_with_cancelled_root_task) do
+    appeal = create(:appeal, :with_post_intake_tasks)
+    create(:bva_dispatch_task, :completed, appeal: appeal)
```

Comment:
To make it more realistic:
```suggestion
    create(:bva_dispatch_task, :completed, parent: appeal.root_task)
```